docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -251,7 +251,7 @@ When a LDAP user logs in to YugabyteDB Anywhere, the system handles role assignm
 
 - If group mapping is enabled:
 
-  - If YugabyteDB Anywhere can obtain a valid role from the LDAP server (from either `yugabytePlatformRole` or group mappings), it assigns the role to the user. If both are defined, the union of roles is assigned. You can't subsequently [change the role for this user](../anywhere-rbac/#create-modify-and-delete-roles) in the **Access Management** tab.
+  - If YugabyteDB Anywhere can obtain a valid role from the LDAP server (from either `yugabytePlatformRole` or group mappings), it assigns the role to the user. If both are defined, the union of roles is assigned. You can't subsequently change the role for this user in the **Access Management** tab.
```

Comment:
I'm just asking: do we no longer have a link for this?

Reply:
there is a link a little earlier